docs/batches-and-continuations.md

```
@@ -358,13 +358,16 @@ like `AwaitingContinuation`.
 | `PendingCount`   | int             | Non-terminal members; decremented as members reach terminal state |
 | `SucceededCount` | int             | Members that succeeded (maintained in §4.4)       |
 | `FailedCount`    | int             | Members that failed                               |
-| `CancelledCount` | int             | Members cancelled (incl. cascade)                 |
+| `CancelledCount` | int             | Members cancelled by an explicit action           |
+| `SkippedCount`   | int             | Conditional branches whose trigger was not selected |
 | `StartedAt`      | timestamptz?    | Set when the first member goes `Active`           |
 | `CompletedAt`    | timestamptz?    | Set when `State` becomes terminal                 |
 | `State`          | enum            | `Executing | Succeeded | Failed | Cancelled`      |
```

markdownlint-cli2 (0.23.1)

[warning] Table column count
Expected: 3; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)